sources/packages/api/src/database/entities/institution-user.model.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Entity,
+  JoinColumn,
+  OneToMany,
+  OneToOne,
+  PrimaryGeneratedColumn,
+} from "typeorm";
+import { InstitutionUserAuth } from "./institution-user-auth.model";
+import { Institution } from "./institution.model";
+import { RecordDataModel } from "./record.model";
+import { User } from "./user.model";
+
+@Entity({
+  name: "institution_users",
+})
+export class InstitutionUser extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @OneToOne((type) => User, { eager: true, cascade: true })
+  @JoinColumn({
+    name: "user_id",
+    referencedColumnName: "id",
+  })
+  user: User;
+
+  @OneToOne((type) => Institution, { eager: true, cascade: true })
```

Comment:
I would recommend removing the eager loading and letting the developer decide when it makes sense to retrieve the relational data.

Reply:
Institution user is basically anchor entity connecting dots. So, I believe eager loading will be required here.